pkg/util/timeutil/now_windows.go

```diff
@@ -41,5 +42,5 @@ func init() {
 func Now() time.Time {
```

Comment:
this comment still talks about local time.

Reply:
Done.